agenda/presentation/src/main/java/com/pronoidsoftware/agenda/presentation/detail/components/event/photo/EventDetailPhotos.kt

```diff
@@ -0,0 +1,320 @@
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.pronoidsoftware.agenda.presentation.R
+import com.pronoidsoftware.core.presentation.designsystem.Inter
+import com.pronoidsoftware.core.presentation.designsystem.LocalSpacing
+import com.pronoidsoftware.core.presentation.designsystem.PlusIcon
+import com.pronoidsoftware.core.presentation.designsystem.TaskyBlack
+import com.pronoidsoftware.core.presentation.designsystem.TaskyGray
+import com.pronoidsoftware.core.presentation.designsystem.TaskyTheme
+
+@Composable
+fun EventDetailPhotos(
+    photos: List<PhotoId>,
+    arePhotosFull: Boolean,
+    editEnabled: Boolean,
+    onAddClick: () -> Unit,
+    onOpenClick: () -> Unit,
+    onEditClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    if (photos.isEmpty()) {
+        NoPhotosCarousel(
+            editEnabled = editEnabled,
+            onAddClick = onAddClick,
+            modifier = modifier,
+        )
+    } else {
+        PhotoCarousel(
+            photos = photos,
+            arePhotosFull = arePhotosFull,
+            editEnabled = editEnabled,
+            onAddClick = onAddClick,
+            onOpenClick = onOpenClick,
+            onEditClick = onEditClick,
+            modifier = modifier,
+        )
+    }
+}
+
+@Composable
+private fun NoPhotosCarousel(
+    editEnabled: Boolean,
+    onAddClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val spacing = LocalSpacing.current
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(spacing.noPhotosRowHeight)
+            .background(MaterialTheme.colorScheme.surfaceVariant)
+            .then(
+                if (editEnabled) {
+                    Modifier.clickable {
+                        onAddClick()
+                    }
+                } else {
+                    Modifier
+                },
+            ),
```

Comment:
The clickable modifier has an enabled attribute:
```
clickable(enabled = editEnabled) {
    onAddClick()
}
```

Reply:
Thanks, this is much more readable